lib/sidekiq/testing.rb

```diff
@@ -323,6 +323,15 @@ def drain_all
       end
     end
   end
+
+  module TestingExtensions
+    def jobs_for(klass)
+      jobs.select { |job| YAML.load(job["args"][0])[0] == klass }
```

Comment:
I would add a quick check to prune obviously wrong values and avoid parsing the entire YAML if not necessary, e.g.

```ruby
jobs.select { |job| job["args"].index(klass.to_s) && YAML.load(job["args"][0])[0] == klass }
```

What do you think?

Reply:
Good idea. But `job["args"]` is an array, so changed code a bit.